### DISCUSSION TITLE
Country settings in FDPP

### DISCUSSION THREAD

**Discussion by andrewbird** [dosemu2/fdpp#192] [Ideas]
I'm about to write some dosemu2 tests to checkout the state of FreeDOS kernel country settings, as there seems to be an issue https://github.com/FDOS/country/issues/5#event-5831065914 there.  I recently made the https://github.com/FDOS/country repo a submodule of kernel and moved the default settings (those that are used when country.sys has not been loaded) to there so that people may update them in one place. I'm thinking that perhaps FDPP could do better than this and have a single table for country.sys always built in, just like you did for share.com. Would it be possible to do this in FDPP without a hit to DOS memory? Are there any downsides?

**Comment by stsp** [comment]
Embedding country default while
still allowing to use country.sys, is
perfectly fine with me.
But looking into the repo you pointed,
it seems to be in asm, whereas share
was in C.
Happy porting. :)

**Reply by andrewbird** [reply]
Mmm, perhaps more work than I envisaged.
